### PR TITLE
net_flow_graph: Data as throughput and a throughput filter.

### PR DESCRIPTION
**Pull request by oazizi000**

### CODE REVIEW

File: pxl_scripts/px/net_flow/net_flow.pxl [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    # Filter out any non k8s sources.
+    df = df[df['pod'] != '']
+
+    # Use aggregate to pick the latest sample for any given client-server pair.
+    # We do this by picking the max of the stats, since they are all counters.
+    df = df.groupby(['pod', 'upid', 'remote_addr']).agg(
+        bytes_sent=('bytes_sent', px.max),
+        bytes_recv=('bytes_recv', px.max),
+    )
+    df.bytes_total = df.bytes_sent + df.bytes_recv
```
Comment (philkuz) [inline pxl_scripts/px/net_flow/net_flow.pxl:29]: I would probably switch everything to use the `df.colname` syntax, looks cleaner to me

**Review by philkuz — CHANGES_REQUESTED** [review]

**Comment by oazizi000** [comment]
Original push was messed up. This is now ready for review.

**Review by philkuz — COMMENTED** [review]
what unit is throughput filter supposed to be? bytes/s?

**Review by philkuz — APPROVED** [review]
i would update the description of the vis variable with the units

**Review by zasgar — APPROVED** [review]

**Pull request merged**